front/app/containers/ProjectsShowPage/timeline/PhaseDescription.tsx

```diff
@@ -20,14 +21,19 @@ import usePhaseFiles from 'api/phase_files/usePhaseFiles';
 // utils
 import { isCurrentPhase } from 'api/phases/utils';
 
-const Container = styled.div`
+// typings
+import { IPhases } from 'api/phases/types';
+
+const Container = styled.div<{ hasBottomMargin: boolean }>`
   padding: 30px;
   padding-bottom: 35px;
   ${defaultCardStyle};
 
   ${media.phone`
     padding: 20px;
   `}
+
+  margin-bottom: ${(props) => (props.hasBottomMargin ? '50px' : '0px')};
```

Comment:
I think this is a little bit too large of a space on mobile - could we make this margin smaller for phones? :)

![image](https://github.com/CitizenLabDotCo/citizenlab/assets/33987955/ec0364c4-f3d9-4963-9350-4f6b660bf83f)